### ISSUE TITLE
Pro-Vice Chancellor Spawning

### ISSUE THREAD

**Issue by luvies** [Gorea235/Uni-UniversityDominationA3#3]
We need to spawn and place the PVC in a random, unclaimed sector, and after the mini-game is complete.

It could be stored in the sector directly, or separately by the overall game state. The former might be easier to work with (and might work well with the Landmark class)

**Comment by asyncmeonov** [comment]
A bit of clarification from the requirements: to quote F2 
> PVC should not spawn at a landmark or in a sector occupied by a unit.

are the only restrictions so a PVC _should_ be able to spawn even on a conquered sector as long as there is no landmark and/or unit on it. This make sense in the current game setting as the map is very small and could be conquered entirely for about 4-5 turns (taken all 4 players are in game) which would mean that after that no PVC will ever be spawned. A simple suggestion to counter that would be:

- delay PVC spawning by _n_ (tbd) turns after being captured
- PVC discovery is triggered not by stepping on the sector where he is, but by conquering it 
  - possible problem: the player whose territory the PVC has spawned on would never discover it
  - possible mitigation: after discovery, realocate PVC on the set of spaces that are {(allPlayers/lastFoundPlayer) U unconqueredSectors} wich would bring balance, as only the player who previously discovered the PVC (the _lastFoundPlayer_) would not be able to immediately stumble upon him.

**Comment by luvies** [comment]
Taking the requirements as-is, I believe that we are allowed to spawn it on conqured sectors, as long as there isn't a unit or landmark on it. Any further restrictions is entirely down to us.

I think a spawning delay might help game pacing a lot, so we should probably add that. Also, restricting the spawning to only sectors that are unclaimed or are not claimed by the player who collected it last is probably a very good thing to do, but maybe only if we require re-conquering.

The issue currently, is that units are allocated at a too slow of a pace to enable constant re-conquering that this system would need. If we allowed players to spread out units (see #9), and increased allocation rate, it might make this kind of system much more workable. (It might also be better if we expanded the map, as that would allow larger games that support the re-conquering gameplay much better)
